src/test/java/uk/gov/di/ipv/lambda/AuthorizationHandlerTest.java

```diff
@@ -56,11 +55,16 @@ public void shouldReturnAuthResponseOnSuccessfulOauthRequest() throws Exception
         params.put("client_id", "12345");
         params.put("response_type", "code");
         params.put("scope", "openid");
+        params.put("scope", "openid");
```

Comment:
This line is duplicated :)

Reply:
Ah yeah good spot. I'll remove this. Cheers